site/packages/aa-ethers/provider-adapter/connectToAccount.md

```diff
@@ -14,7 +14,7 @@ head:
 
 # connectToAccount
 
-`connectToAccount` is a method on `EthersProviderAdapter` that you can optionally call to connect the provider to an account and returns a `AccountSigner`. This enables the returned `AccountSigner` to leverage the provider when signing messages, UserOperations, and transactions for a smart account using the owner account.
+`connectToAccount` is a method on `EthersProviderAdapter` that you can optionally call to connect the provider to an account and returns a `AccountSigner`. This enables the returned `AccountSigner` to leverage the provider when signing messages, UserOperations, and transactions for a smart account using the connected EOA signer account.
```

Comment:
a signer isn't required to be an EOA. just most of today's signers are, but a signer could also be a passkey signer for example

Reply:
good point.